### PR TITLE
Bootstrap location snippet

### PR DESCRIPTION
**Pull request by cjf-fuller**
- Update on gitignore for visual studio code usage
- custom css for the map canvas
- location sample snippet (not styled yet)

### CODE REVIEW

File: fold/mockup/index.html [hunk 1]
```diff
@@ -1,23 +1,60 @@
...
+    crossorigin="anonymous">
+
+  <title>Fold</title>
+</head>
+
+<body>
+
+  <!-- we put HTML here -->
+  <!--Begin Location Snippet-->
+  <div class="container">
```
Comment (JoeClarkMcGhee) [inline fold/mockup/index.html]: As the location part of the site will be found by clicking on contact in the nav bar I think that all of your changes will need to live in a separate HTML file called, for example, `contact.html`.  The `index.html` page will just have the upcoming and past listings.

File: fold/mockup/index.html [hunk 2]
```diff
@@ -1,23 +1,60 @@
...
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="css/main.css">
+
+  <!-- Fonts -->
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.10/css/all.css" integrity="sha384-+d0P83n9kaQMCwj8F4RJB66tzIwOKmrdb46+porD/OvrJ+37WqIM7UoBtwHO6Nlg"
+    crossorigin="anonymous">
```
Comment (JoeClarkMcGhee) [inline fold/mockup/index.html]: Do you know what additional font you are using by importing this?

File: fold/mockup/index.html [hunk 3]
```diff
@@ -43,7 +43,7 @@ <h4 class="pt-2"><i class="fas fa-map-marker-alt" style="color:#000"></i>  Addre
       <div class="col-sm-8">
         <div class="mapouter">
           <div class="gmap_canvas"><iframe id="gmap_canvas" src="https://maps.google.com/maps?q=FOLD%2C%20gillian%20house%2C%20Stephenson%20Street%2C%20London&t=&z=13&ie=UTF8&iwloc=&output=embed"
-              width="100%" height="420" frameborder="0" style="border:0" allowfullscree></div><style></style>
+              width="100%" height="420" frameborder="0" style="border:0" allowfullscree></div>
```
Comment (JoeClarkMcGhee) [inline fold/mockup/index.html]: I think that we should squash this commit with the previous.

>Steps
1. Soft reset the head to the previous commit `git reset --soft 4167b7b144ef552fdab8bbd70c98d22fd38d3095`
2. Add the file to the staging area `git add fold/mockup/index.html`
3. Amend the commit to include the changes in your staging area `git commit --ammend --no-edit`
4. Force push the branch `git push -f --set-upstream origin bootstrap-location-snippet`

File: .gitignore [hunk 4]
```diff
@@ -16,6 +16,6 @@ fold/db.sqlite3
 *.iml
 Pipfile.lock
 
-# VS Code artifacts
+# VS Code user settings
 
-*.vscode
\ No newline at end of file
+.vscode/
```
Comment (JoeClarkMcGhee) [inline .gitignore]: In future, I think that any changes to the git ignore should be contained within there own PR. There is no point moving this one out though!

Comment (cjf-fuller) [reply .gitignore]: Yeah good point, I’ll do that going forward

File: fold/mockup/index.html (hunk 2 above)
Comment (cjf-fuller) [reply fold/mockup/index.html]: That’s the little like icon thing by address. I’ll check if bootstrap already calls it though

Comment (cjf-fuller) [reply fold/mockup/index.html]: had a quick look, looks as though it's not included with bootstrap, 
gives access to these icons in html class setting https://fontawesome.com/icons
update the release to the latest now

File: fold/mockup/index.html (hunk 3 above)
Comment (cjf-fuller) [reply fold/mockup/index.html]: steps worked, handy tip, never knew how to squash!

File: fold/mockup/index.html (hunk 1 above)
Comment (JoeClarkMcGhee) [reply fold/mockup/index.html]: We have decided to go with an SPA style site so we will only have one `index.html` file and one `blog_post.html` file.

**Comment by JoeClarkMcGhee** [comment]
When I checked out this branch and viewed the changes in the browser I noticed that the maps and other bits did not rescale properly when viewed on a mobile device. Take a look at [this](https://getbootstrap.com/docs/4.0/components/navbar/l) example of a page that resizes when you change the browsers width it also works on phones. I'm not sure how to fix this one but I'm sure that there will be a Bootstap class that will be able to handle this.

**Comment by cjf-fuller** [comment]
@JoeClarkMcGhee Okay cool, do you mean the change when the map goes below the address line on a small screen? I have set up the bootstrap column class to use the flexbox responsive grid system and set it to small, but could change it to extra small.

**Comment by JoeClarkMcGhee** [comment]
@cjf-fuller I dont think that we need to be mega fused about it look ok when the screen is resized to something ridiculously small, the main concern is that the page looks OK on mobiles

**Comment by cjf-fuller** [comment]
okay gotcha, below is the layout on an iphone 6, are you thinking shift the text a bit so it is more central?

image

**Comment by JoeClarkMcGhee** [comment]
No, I think that is fine now :+1:, it looks nicely contained.
Push up the changes and I'll take a look.

**Comment by cjf-fuller** [comment]
I didn't change anything, so I'm worried there is something I am missing, have you got a pic of it looking bad? need to fix some merge conflicts so will push them out aswell

**Comment by JoeClarkMcGhee** [comment]
OK, let me know when I can pull it down again. You might need to rebase on top of master?

**Comment by cjf-fuller** [comment]
sweet, fixed the merge conflicts, ready to merge when you are

**Comment by JoeClarkMcGhee** [comment]
There are a few issues with how it sits under the header. Take a look at the snap

Selection_018

**Comment by cjf-fuller** [comment]
ah of course, sorry forgot to add in the padding to avoid that. have committed that now,

ref: https://stackoverflow.com/questions/10336194/twitter-bootstrap-top-nav-bar-blocking-top-content-of-the-page?answertab=votes#tab-top

**Comment by JoeClarkMcGhee** [comment]
I'm not sure you pushed any changes?

**Comment by cjf-fuller** [comment]
forgot to press enter! up now, my bad

**Comment by cjf-fuller** [comment]
just need to adjust the px height of the padding if we change the height of the navbar

**Comment by JoeClarkMcGhee** [comment]
Great Job man. Merge that puppy!

**Pull request merged**